integers/overflow/07_simple_cross_file.c

```diff
@@ -13,7 +13,12 @@ void integer_overflow_cross_file(bool a, bool b) {
     }
 #endif
 
-    val = choose_int(2, INT_MAX / 3 - 1, INT_MAX / 2 - 1);
+    if (!a && !b) {
+        val = choose_int(2, INT_MAX / 2 - 1, 0);
+    }
+    else
+        val = choose_int(2, INT_MAX / 3 - 1, INT_MAX / 2 - 1);
+
```

Comment:
I think that this is a better fix as I think that `a` should have been the third argument: 
```suggestion
    val = choose_int (INT_MAX / 3 - 1, INT_MAX / 2 - 1, a);
```
See https://github.com/JMoerman/JM2018TS/blob/ce53f08350648a878772f844e05ef5c56cb2d004/common/common.c#L37

Reply:
Yes I totally get it, it does seem better that way !